### ISSUE TITLE
better PDF filtering 

### ISSUE THREAD

**Issue by LindsayYoung** [18F/openFEC#901]
Add more complex date filtering 

Right now we are just filtering since 1994, this will make some bad links for Senators. 

As Paul said:
```
Older paper reports are availabe with the following limitations:

PAC and party reports since 1993 
House committee reports since May 1996
Senate filings since May 2000

http://www.fec.gov/finance/disclosure/help_candidate_committee_viewer.shtml

```
@dcpcc1967  what about Presidential?

**Comment by dcpcc1967** [comment]
1993 for presidential filings

**Comment by jmcarp** [comment]
Fixed in #928.

**Comment by PaulClark2** [comment]
I am out of the office until 06/16/2015.

I'm on leave and will respond to your e-mail when I return.  If you need
immediate assistance please contact Jeff Chumley, jchumley@fec.gov, or Wei
Luo, wluo@fec.gov.



Note: This is an automated response to your message  "Re: [openFEC] better
PDF filtering  (#901)" sent on 6/15/2015 11:00:02 AM.

This is the only notification you will receive while this person is away.
